tests/framework/extensions/rke1/nodepools/nodepools.go

```diff
@@ -87,7 +87,7 @@ func NodePoolSetup(client *rancher.Client, nodeRoles []NodeRoles, ClusterID, Nod
 		nodePoolConfig.Etcd = roles.Etcd
 		nodePoolConfig.Worker = roles.Worker
 		nodePoolConfig.Quantity = roles.Quantity
-		nodePoolConfig.HostnamePrefix = "auto-rke1-" + strconv.Itoa(index)
+		nodePoolConfig.HostnamePrefix = "auto-rke1-" + strconv.Itoa(index) + ClusterID
```

Comment:
Just needs a rebasing and this will be fixed, but this has been updated in a separate RKE1 PR.